webpack/assets/javascripts/react_app/components/PF4/InlineEdit/InlineTextInput.js

```diff
@@ -0,0 +1,50 @@
+import React, { createRef } from 'react';
+import { TextInput, TextArea } from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+import { useSubmitOnEnter } from '../../../common/hooks/useSubmitOnEnter';
+
+const InlineTextInput = ({
+  textArea,
+  attribute,
+  value,
+  setInputValue,
+  onSubmit,
+}) => {
+  const inputRef = createRef(null);
+  useSubmitOnEnter(inputRef, onSubmit);
+
+  const inputProps = {
+    value,
+    onChange: v => setInputValue(v),
+    onFocus: () => inputRef?.current && inputRef.current.select(),
```

Comment:
You can use optional chaining with function calls:
```suggestion
    onFocus: () => inputRef?.current?.select?.(),
```